app/webpacker/components/Tickets/TicketComments.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import { Button, Comment, Header } from 'semantic-ui-react';
+import { useQuery } from '@tanstack/react-query';
+import { DateTime } from 'luxon';
+import TicketCommentCreate from './TicketCommentCreate';
+import getComments from './api/getComments';
+import Loading from '../Requests/Loading';
+import Errored from '../Requests/Errored';
+import '../../stylesheets/semantic/components/comment.min.css';
+
+export default function TicketComments({ ticketId, currentStakeholder }) {
+  const [createComment, setCreateComment] = useState(false);
+
+  const {
+    data: { comments } = { comments: [] }, isFetching, isError, refetch,
+  } = useQuery({
+    queryKey: ['ticket-comments', ticketId],
+    queryFn: () => getComments({ ticketId }),
```

Comment:
```suggestion
    queryKey: ['ticket-comments', ticketId],
    queryFn: () => getComments({ ticketId }),
    select: (data) => data.comments,
```

Reply:
see above. Note that I wrote this code blindly on GH, without actually testing whether it covers your use-case. Please treat this suggestion with caution :)